src/software/ai/passing/pass.cpp

```diff
@@ -40,14 +38,14 @@ double Pass::speed() const
     return pass_speed_m_per_s;
 }
 
-Timestamp Pass::startTime() const
+Pass Pass::fromPassArray(Point passer_point, const std::array<double, 3>& array)
 {
-    return pass_start_time;
+    return Pass(passer_point, Point(array.at(0), array.at(1)), array.at(2));
 }
 
-Timestamp Pass::estimateReceiveTime() const
+std::array<double, 3> Pass::toPassArray() const
```

Comment:
```suggestion
std::array<double, NUM_PARAMS_TO_OPTIMIZE> Pass::toPassArray() const
```
and same for all array definitions

Reply:
fixed